src/Fixer/ConstantNotation/NativeConstantInvocationFixer.php

```diff
@@ -225,6 +212,59 @@ protected function createConfigurationDefinition()
                 ->setAllowedValues([$constantChecker])
                 ->setDefault(['null', 'false', 'true'])
                 ->getOption(),
+            (new FixerOptionBuilder('scope', 'Only fix constant invocations that are made within a namespace or fix all.'))
+                ->setAllowedValues(['all', 'namespaced'])
+                ->setDefault('all')
+                ->getOption(),
         ]);
     }
+
+    private function fixConstantInvocations(Tokens $tokens, $start, $end)
```

Comment:
If a file with multiple namespaces is fixed this method might be called multiple times (based on the new config option). 
The `$useDeclarations` found and the token analyzer (created later on) are based on the complete tokens collection, these can be reused as long as no tokens are inserted.

Reply:
but this method inserts tokens.

Actually, a better way (for both this fixer and the function invocation one) would be to analyze use statements only for the current namespace anyway. but the use statement analyzer does not support that currently.